pkg/visitors/dxecleaner.go

```diff
@@ -126,11 +126,13 @@ func parseBlackList(fileName, fileContents string) (string, error) {
 		if line == "" || strings.HasPrefix(line, "#") {
 			continue
 		}
-		_, err := regexp.Compile(line)
+		// skip multiple words, We need this for linuxboot file parsing
+		fw := strings.Split(line, " ")[0]
+		_, err := regexp.Compile(fw)
 		if err != nil {
-			return "", fmt.Errorf("cannot compile regex %q from blacklist file %q: %v", line, fileName, err)
+			return "", fmt.Errorf("cannot compile regex %q from blacklist file %q: %v", fw, fileName, err)
```

Comment:
consider printing out the line here. In the for, for i, line := range
then you can print out the line number, the line, and the first element. It's helpful.

Reply:
Good idea, Thanks!

Reply:
Done